src/System.Security.Cryptography.Xml/tests/XmlDecryptionTransformTest.cs

```diff
@@ -134,6 +134,7 @@ public void LoadInnerXml_XmlValidUriForExcept()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "https://github.com/dotnet/corefx/issues/18690")]
```

Comment:
Also, is the problem that these tests are reusing `transform`, and it's getting contaminated by another test? (reusing the object being tested isn't very unit-y)